### DISCUSSION TITLE
Release 2.1?

### DISCUSSION THREAD

**Discussion by essen** [ninenines/ranch#331] [General]
@Maria-12648430 @juhlig I think everything is ready for 2.1 unless I'm forgetting something. All good?

**Comment by Maria-12648430** [comment]
As far as I can tell, all is good and nice and shiny 🥰

**Comment by juhlig** [comment]
Same here, no objections :)

**Comment by essen** [comment]
OK I finish preparing and then release tomorrow.

**Reply by Maria-12648430** [reply]
Yay 😎

**Comment by essen** [comment]
All done. Thanks! Locking the topic.
